build/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
Change short name and name to something more reasonable.

Reply:
@NJaku01 Can you take care of it so that it is done in general for the build?

Reply:
I have adapted it in my fork, the changes will be available in the build with the next release